### ISSUE TITLE
Disable Shuffling of Data

### ISSUE THREAD

**Issue by justin-chau** [justin-chau/genetic-hyperparameters#3]
https://keras.io/preprocessing/image/
